reporting_app/static/column_formatter.js

```diff
@@ -23,9 +23,15 @@ function string_formatter(cell_data, fmt, row){
     if (cell_data instanceof Array) {
         cell_data.sort();
     } else if (cell_data instanceof Object) {
-        // convert, e.g, {'this': 0, 'that': 1, 'other': 2} to ['this: 0', 'that: 1', 'other: 2']
+        // convert, e.g, {'this': 0, 'that': 1, 'other': 2} to ['other: 2', 'that: 1', 'this: 0']
```

Comment:
@mwhamgenomics Why has the order of the pairs in the output changed?

Reply:
See line 29:
```javascript
keys.sort()
```
This makes the order of the dropdown deterministic/consistent between rows.

Reply:
I see, so the example now represents how it would be sorted.